### DISCUSSION TITLE
SuperSlicer "Cutting corners" Feature

### DISCUSSION THREAD

**Discussion by Fusiable** [OrcaSlicer/OrcaSlicer#1744] [Ideas]
The cutting corners feature from SuperSlicer may further improve bulging corners, with the combination of a good pressure advance value. Not completely certain on this one, but would be great to have a nightly in order to run some tests. I'm not sure if its even possible to port that over?

**Comment by RandomGgames** [comment]
I would love to see this. I have a fairly well tuned and mostly stock Ender 3 (other than a BTT board) which I've been trying to battle rounded corners with for years at this point. Having a cutting corner feature would be a life saver. I've started loving this slicer a lot, and adding this would just add to the list.

**Comment by Eldenroot** [comment]
+1
